ts/output/common/Wrappers/scriptbase.ts

```diff
@@ -277,7 +302,7 @@ export function CommonScriptbaseMixin<
      * @return {boolean}  True if the base is an mi, mn, or mo (not a largeop) consisting of a single character
      */
     public isCharBase(): boolean {
-      let base = this.baseChild;
+      let base = this.getRealBaseChild();
```

Comment:
This is ruining the placement in sub/sup expressions. So should be moved to a specialist method in the `mmultiscripts` class.

Reply:
Done